Release Saved Insights

mariusandra opened the pull request:
## Changes

- Removes the saved insights flag
- Removes the insight history page and related logic
- Adds a redirect in case someone still uses the /insights?insight=HISTORY url

## How did you test this code?

- Tried clicking different pages in the interface. Everything worked.

mariusandra committed:
release saved insights feature

mariusandra committed:
remove viewType and add fallback route

mariusandra committed:
remove test for insights history

mariusandra committed:
fix test

mariusandra committed:
redirect root to saved insights page

mariusandra committed:
fix tests

mariusandra committed:
more test fixes

mariusandra committed:
Merge branch 'master' into insights-are-saved

Twixes commented on frontend/src/scenes/scenes.ts:
```diff
@@ -130,7 +130,7 @@ export const sceneConfigurations: Partial<Record<Scene, SceneConfig>> = {
 
 export const redirects: Record<string, string | ((params: Params) => string)> = {
-    '/': urls.insights(),
+    '/': urls.savedInsights(),
```
This change is interesting. I'm a bit sad that exploration is now less of a default, though this is probably better for the collaboration angle.

paolodamico replied:
We will probably want to test this a bit more ongoingly, I'm particularly worried it will negatively impact new users who just signed up to an account. Our empty state here needs to be killer. Thoughts @clarkus ?

Twixes reviewed (commented):
Sidebar item highlight is no longer working for the insight scene.
![Zrzut ekranu 2021-11-9 o 21 01 48](https://user-images.githubusercontent.com/4550621/140996636-9556440a-351b-4641-a675-1dc3cd97e681.png)

mariusandra committed:
Merge branch 'master' into insights-are-saved

mariusandra committed:
fix more tests

mariusandra committed:
try a different fix

mariusandra committed:
fix sidebar iteem

mariusandra commented:
Fixed the sidebar item and fixed the tests. Should be clear for merge.

Indeed, making `urls.savedInsights()` the default for `/` is a new change here, which I thought we had agreed on doing. In any case, we can surely revisit this and revert that change if needed.

paolodamico reviewed (approved):
lgtm!
